revolv/project/fixtures/category.json

```diff
@@ -0,0 +1,44 @@
+[
+{
```

Comment:
Let me know what your take on loading categories through fixtures is. I can move category creation to code if necessary.